_posts/2022-09-20-Project-Sabine-Low-cost-DIY-underwater-modem-using-COTS-components-and-Unet-audio.md

```diff
@@ -0,0 +1,326 @@
+---
+layout: post
+comments: true
+title: Low-cost DIY underwater modem using COTS components and Unet audio
+date: 20/9/2022
+author: Manu Ignatius, Sukanta K. Hazra
+categories: howto
+feature-img: "assets/img/stdma/stdma-unet.jpg"
+thumbnail: "assets/img/stdma/stdma-unet.jpg"
+tags: [howto, unetaudio, unetstack, modems, phy, janus, sabine]
+---
+
+## Introduction
+In the article titled [Converting your laptop into a JANUS modem using Unet audio](https://blog.unetstack.net/converting-your-laptop-into-a-janus-modem-using-unetaudio), we saw how UnetStack, a Software-Defined Open Architecture Modem (SDOAM), can be used to turn your laptop into an acoustic modem. A true SDOAM can easily be extended to run on various hardware platforms with different types of physical layers (e.g. Acoustic, optical, RF) with minimal efforts to address different use cases. In this article, we will show how you can build a low-cost (< USD 300) , DIY underwater acoustic modem using only Commercial Off-The-Shelf (COTS) components and Unet audio. The goal of this article is to demonstrate the approach and basic steps one can use to build a low cost acoustic modem using Unet audio. We encourage the user to build on these principles to build create versions of the DIY modem.
+
+## Motivation and Approach
+Most commercially available modems are beyond the reach of a technology enthusiast or a hobbyist underwater roboticist, due to their price point. However, applications such as DIY underwater vehicles (e.g. for [The Singapore AUV Challenge](https://sauvc.org/)), low-cost swarms, etc. need low-cost underwater modems to realize their true potential. While there have been projects like the [AHOI modem](https://dl.acm.org/doi/10.1145/3376921), [SeaModem](https://ieeexplore.ieee.org/document/7271721), [Seatrac](https://ieeexplore.ieee.org/document/7271578), all of them has a hardware component associated with it. This means the user needs to spend time building the hardware, which can be cumbersome and time-consuming. Adding additional hardware to a micro-AUV can be challenging due to space and power limitations. Ensuring long-term support for these can also be tricky, depending on the creator's other commitments.
```

Comment:
- `all of them has a hardware component associated with it` => `all of them have custom hardware associated with them`